helmfile/charts/log-manager/templates/secret.yaml

```
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "log-manager.fullname" . }}-s3-credentials
+  labels:
+    {{- include "log-manager.labels" . | nindent 4 }}
+stringData:
+  s3cfg.ini: |
+    host_base         = {{ .Values.s3.regionEndpoint }}
+    {{- if .Values.s3.forcePathStyle }}
+    host_bucket       = {{ .Values.s3.regionEndpoint }}
+    {{- else }}
+    host_bucket       = %(bucket)s.{{ .Values.s3.regionEndpoint }}
```

Does this work if region endpoint contains https/http? 
```suggestion
    host_bucket       = %(bucket)s.{{ .Values.s3.regionEndpoint }}
```

---

Good catch, it won't.